Description: Edit menus from gnome(or $XDG_MENU_PREFIX)-applications.menu
  instead of applications.menu
Author: Iain Lane <laney@debian.org>
Forwarded: not-needed
Bug-Debian: http://bugs.debian.org/676137

Index: b/Alacarte/MenuEditor.py
===================================================================
--- a/Alacarte/MenuEditor.py
+++ b/Alacarte/MenuEditor.py
@@ -24,6 +24,8 @@
 
 class MenuEditor(object):
     def __init__(self, name='applications.menu'):
+        menu_prefix = os.getenv('XDG_MENU_PREFIX') or 'gnome-'
+        name = menu_prefix + name
         self.name = name
 
         self.tree = GMenu.Tree.new(name, GMenu.TreeFlags.SHOW_EMPTY|GMenu.TreeFlags.INCLUDE_EXCLUDED|GMenu.TreeFlags.INCLUDE_NODISPLAY|GMenu.TreeFlags.SHOW_ALL_SEPARATORS|GMenu.TreeFlags.SORT_DISPLAY_NAME)
